src/components/NavBar/navBar-style.css

```diff
@@ -0,0 +1,8 @@
+.active {
+  font-weight: bold;
+}
+
+a {
```

Comment:
Would it be possible to scope this styling to a CSS className so that we avoid polluting the global namespace? We're not 100% sure that we want all `a` tags in the app to not have `text-decoration`, right?